src/webview.rs

```diff
@@ -136,6 +152,59 @@ impl WebViewBuilder {
         self
     }
 
+    /// Set the RPC handler.
+    pub(crate) fn set_rpc_handler(mut self, proxy: WindowProxy, handler: Arc<RpcHandler>) -> Self {
+        let js =
+            r#"
+            function Rpc() {
+                this._callback = '__rpc__';
+                this._promises = {};
+
+                // Private internal function called on error
+                this._error = (id, error) => {
+                    if(this._promises[id]){
+                        this._promises[id].reject(error);
+                        delete this._promises[id];
+                    }
+                }
+
+                // Private internal function called on result
+                this._result = (id, result) => {
+                    if(this._promises[id]){
+                        this._promises[id].resolve(result);
+                        delete this._promises[id];
+                    }
+                }
+
+                // Call remote method and expect a reply from the handler
+                this.call = function(method) {
+                    const id = Math.floor(Math.random() * Number.MAX_SAFE_INTEGER);
+                    const params = Array.prototype.slice.call(arguments, 1);
```

Comment:
Does this mean it only takes one argument to RPC handler?

Reply:
Nope, that is all remaining arguments after the `method` argument.